tests/members/test_members_services.py

```diff
@@ -68,6 +69,33 @@ def test_add_denied(
         data
     )
 
+@pytest.mark.parametrize("actor,role", [
```

Comment:
Using `parametrize` is very nice and clean. However, it has the downside of calling the function once per use case (in this case 7 times). When the fixtures used by the test function are of scope "module" it is okay to use `parametrize` since fixtures will only be created/destroyed once. However, this test has `members` which creates serveral users (4). Those users are being created/removed each time (i.e. 28 calls + db session destroy/rollback).

I merely want to avoid ending up in a situation like rdm-records, where tests take 10+minutes to run.

Can this test be refactored with a for loop inside the test function?